hack/aks/Makefile

```diff
@@ -185,13 +213,14 @@ overlay-up: rg-up overlay-net-up ## Brings up an Overlay AzCNI cluster
 	@$(MAKE) set-kubeconf
 
 swift-byocni-up: rg-up swift-net-up ## Bring up a SWIFT BYO CNI cluster
+	@$(MAKE) public-ipv4
 	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
 		--auto-upgrade-channel $(AUTOUPGRADE) \
 		--node-os-upgrade-channel $(NODEUPGRADE) \
 		--kubernetes-version $(K8S_VER) \
 		--node-count $(NODE_COUNT) \
 		--node-vm-size $(VM_SIZE) \
-		--load-balancer-sku standard \
```

Comment:
Inconsistent removal of `--load-balancer-sku`. Does it need to be `standard` or can we keep it as `basic`?

Reply:
can be basic, I meant to remove `--load-balancer-sku standard` for `nodesubnet-byocni-nokubeproxy-up` since I'm adding `--load-balancer-outbound-ips <publicip v4>,<publicip v6>`

Reply:
Does this implicitly set every cluster to standard for the load balancer sku?